src/components/CampaignLocations/CampaignLocationsScreen.tsx

```diff
@@ -148,20 +165,16 @@ export const CampaignLocationsScreen: FunctionComponent<NavigationProps> = ({
               <AppText style={styles.selectCampaignHeader}>
                 Select campaign
               </AppText>
-              {Object.entries(authCredentials).map(
-                ([key, credentials], idx) => (
-                  <View key={key} style={styles.campaignLocationWrapper}>
-                    <CampaignLocationsListItem
-                      {...credentials}
-                      name={
-                        allCampaignConfigs[key]?.features?.campaignName ||
-                        `Campaign ${idx + 1}`
-                      }
-                      onPress={() => navigateToCampaignLocation(credentials)}
-                    />
-                  </View>
-                )
-              )}
+
+              {sortedAuthCredentialsWithCampaignName.map((obj, idx) => (
+                <View key={obj.key} style={styles.campaignLocationWrapper}>
+                  <CampaignLocationsListItem
+                    {...obj}
+                    name={obj.name || `Campaign ${idx + 1}`}
+                    onPress={() => navigateToCampaignLocation(obj)}
+                  />
+                </View>
+              ))}
```

Comment:
It's better practice to name the variable rather than using a generic `obj` variable. Notice in the original it's `credentials`

Reply:
renamed obj as credentials

Reply:
changed branch title to feat: Ordering multiple campaigns in alphabetic order